contracts/governance/FeeSharingCollector/FeeSharingCollector.sol

```diff
@@ -56,14 +56,14 @@ contract FeeSharingCollector is
     using SafeERC20 for IERC20;
 
     address constant ZERO_ADDRESS = address(0);
-    address public constant RBTC_DUMMY_ADDRESS_FOR_CHECKPOINT =
-        address(uint160(uint256(keccak256("RBTC_DUMMY_ADDRESS_FOR_CHECKPOINT"))));
+    address public constant NATIVE_TOKEN_DUMMY_ADDRESS_FOR_CHECKPOINT =
+        address(uint160(uint256(keccak256("NATIVE_TOKEN_DUMMY_ADDRESS_FOR_CHECKPOINT"))));
```

Comment:
Changing the value of this constant is potentially a time bomb waiting to happen, in case this code eventually finds its way to other branches and gets deployed on RSK.

For safety, I would just use the old value here. Possibly hard-coded, in case you don't want to have the string "RBTC" in the contract :P